packages/core-data/src/selectors.js

```diff
@@ -121,6 +121,21 @@ export function getEntity( state, kind, name ) {
 	return find( state.entities.config, { kind, name } );
 }
 
+/**
+ * Checks whether the attribute is a "raw" attribute or not.
+ *
+ * @param {Object} state     Data state.
+ * @param {string} kind      Entity kind.
+ * @param {string} name      Entity name.
+ * @param {string} attribute Attribute name.
+ *
+ * @return {boolean} Is the attribute raw
+ */
+export function isRawAttribute( state, kind, name, attribute ) {
```

Comment:
Does this need to be a public selector?

Reply:
I'd prefer it to be private. The only reason why it's public is that `saveEntityRecord` needs to call it. How about we change the name to `__unstableIsRawAttribute` to fix the widgets bug, and then later remove it as a part of the larger refactoring discussed below?

Reply:
> Or did you just mean to remove it from `core-data` exports?

I think this is what @youknowriad meant. If we can avoid exporting it then I'd prefer that too 🙂